src/tests/Microsoft.PowerFx.Core.Tests.Shared/UserDefinedTypeTests.cs

```diff
@@ -118,11 +118,11 @@ public void TestValidUDTCounts(string typeDefinition, int expectedDefinedTypesCo
         [Theory]
 
         //To test DefinitionsCheckResult.ApplyErrors method and error messages
-        [InlineData("Point = Type({ x: Number, y: Number }); Point = Type(Number);", 1, "ErrNamedType_TypeAlreadyDefined")]
-        [InlineData("X= Type({ f:Number, f:Number});", 1, "ErrNamedType_InvalidTypeDefinition")]
-        [InlineData("B = Type({ x: A }); A = Type(B);", 2, "ErrNamedType_InvalidCycles")]
-        [InlineData("B = Type(B);", 1, "ErrNamedType_InvalidCycles")]
-        [InlineData("Currency = Type({x: Text}); Record = Type([DateTime]); D = Type(None);", 2, "ErrNamedType_InvalidTypeName")]
+        [InlineData("Point := Type({ x: Number, y: Number }); Point := Type(Number);", 1, "ErrNamedType_TypeAlreadyDefined")]
+        [InlineData("X:= Type({ f:Number, f:Number});", 1, "ErrNamedType_InvalidTypeDefinition")]
+        [InlineData("B := Type({ x: A }); A := Type(B);", 2, "ErrNamedType_InvalidCycles")]
+        [InlineData("B := Type(B);", 1, "ErrNamedType_InvalidCycles")]
+        [InlineData("Currency := Type({x: Text}); Record := Type([DateTime]); D := Type(None);", 2, "ErrNamedType_InvalidTypeName")]
         public void TestUDTErrors(string typeDefinition, int expectedErrorCount, string expectedMessageKey)
```

Comment:
>TestUDTErrors [](http://example.com/codeflow?start=20&length=13)

Can you add a case where the expected message key is the new one (ErrNamedType_MissingTypeLiteral)?